src/services/getCharacterById.js

```diff
@@ -0,0 +1,7 @@
+export const getCharacterById = (_id) => {
```

Comment:
You could move combine your fetchs into a single js page for less clutter. You could make APIFetch or something along those lines.